rest-heroes/src/main/resources/application.yml

```diff
@@ -1,6 +1,13 @@
 quarkus:
   application:
     name: rest-heroes
+  log:
+    level: INFO
```

Comment:
I'm not sure why we need to set this? `INFO` is the default.

Reply:
Same